proposal.md

```diff
@@ -0,0 +1,7 @@
+# Proposal 
+
+## Section 1: Motivation and Purpose
+
+>In today’s world, we are overwhelmed by so many choices everywhere. It even applies to things we do for entertainment. For example, if we want to watch a movie, we could easily waste several minutes trying to find a right one. To save those precious minutes, we plan to create an app that in no time can help us find the perfect movie. This app will allow us to search for movies based on their genres, MPAA rating and release years. Based on these settings, the app will show us the top 10 highest grossing movies. We can then interactively pick a movie from that list to see its viewer rating in comparison to the other movies. Briefly, with few clicks, this app will search hundreds of movies to find a perfect movie for us.  
```

Comment:
Personally, it would take me much longer than "several minutes" so I'd feel comfortable  saying "..we could easily waste a lot of time..."

Then to avoid redundancy with "time": "To save some of that precious time, we plan to create an app that ~~in no time~~ can help us **_quickly_** find the perfect movie"

What do you think?

Reply:
Looks good I will make those changes. @sreejithmunthikodu do you have any comments?

Reply:
Nope. Agre with Ryan.